libs/models/src/parserResult.ts

```
@@ -84,11 +84,11 @@ export const positionEndAdd = <T extends TokenPosition>(pos: T, value: number):
 })
 
 const posStart = (values: number[]): number => {
-    const valid = values.filter(n => n >= 0 && !isNaN(n) && isFinite(n))
+    const valid = values.filter(n => n >= 0 && !Number.isNaN(n) && isFinite(n))
```

Biome

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)